### DISCUSSION TITLE
can add new localization po for zh_tw?

### DISCUSSION THREAD

**Discussion by ablegods** [OrcaSlicer/OrcaSlicer#1847] [Ideas]
@SoftFever 
Just like looking up, is it possible to add traditional Chinese localization?

**Comment by SoftFever** [comment]
> @SoftFever Just like looking up, is it possible to add traditional Chinese localization?

Sure thing.
As long as there's a volunteer willing to do the conversion,
I can help add the option.

**Reply by ablegods** [reply]
I'd be happy to assist with this
But how do I provide my converted files?

**Reply by SoftFever** [reply]
You can put the po file in `localization\i18n\zh_TW`
The file name should be `OrcaSlicer_zh_TW.po`

Then create a PR to submit it.
